packages/workbox-window/src/utils/WorkboxEventTarget.ts

```diff
@@ -27,7 +27,7 @@ export class WorkboxEventTarget {
    */
   addEventListener<K extends keyof WorkboxEventMap>(type: K, listener: (event: WorkboxEventMap[K]) => any) {
     const foo = this._getEventListenersByType(type)
-    foo.add(listener);
+    foo.add(listener as ListenerCallback);
```

Comment:
This and the other place where I declare `listener` as being a `ListenerCallback` might be a little sketchy, so your thoughts are appreciated.

The underlying error is because some varieties of `listener` might have a `data` property (the one that wraps a message event), and TypeScript isn't happy about that.

There might be a better remediation here, like changing the `_getEventListenersByType()` definition so that it returns a `Set` with  different signature.